execution_engine/src/storage/global_state/lmdb.rs

```diff
@@ -24,18 +26,19 @@ use crate::storage::{
     },
 };
 
+/// Global state implemented against LMDB as a backing data store.
 pub struct LmdbGlobalState {
-    pub environment: Arc<LmdbEnvironment>,
-    pub trie_store: Arc<LmdbTrieStore>,
-    pub protocol_data_store: Arc<LmdbProtocolDataStore>,
-    pub empty_root_hash: Blake2bHash,
+    pub(crate) environment: Arc<LmdbEnvironment>,
+    pub(crate) trie_store: Arc<LmdbTrieStore>,
+    pub(crate) protocol_data_store: Arc<LmdbProtocolDataStore>,
+    pub(crate) empty_root_hash: Blake2bHash,
```

Comment:
This is always `blake2b(EmptyPointerBlock)` (same for the in-memory); it _could_ just be replaced by a lazy static.

Reply:
It doesn't seem to be _quite_ that simple in this case, and since it's a documentation-oriented ticket I'd prefer not to refactor this at this point.